Sources/DocCDocumentation/DocCDocumentation.docc/customizing-your-documentation-appearance.md

```diff
@@ -0,0 +1,244 @@
+you can check out the latest Open API specification for the it [here][1], which
+documents all valid settings that could be present.
+
+Let's walk through some basic examples of customizing the appearance of
+documentation now.
+
+### Colors
+
+DocC-Render utilizes [CSS custom properties][2] (CSS variables) to define the
+various colors that are used throughout the HTML elements of rendered
+documentation. Any key/value in the `theme.color` object of
+`theme-settings.json` will either update an existing color property used in the
+renderer or add a new one that could be referenced by existing ones.
+
+**Example**
+
+As a simple example, the following configuration would update the background
+fill color for the "documentation intro" element to `"rebeccapurple"` by
+changing the value of the CSS property to `--color-documentation-intro-fill:
+"rebeccapurple"`.
+
+```json
+{
+  "theme": {
+    "color": {
+      "documentation-intro-fill": "rebeccapurple"
+    }
+  }
+}
+```
+
+![A screenshot of the documentation intro element with a purple background color](theme-screenshot-01.png)
+
+**Example**
+
+Note that any valid CSS color value could be used. Here is a more advanced
+example which demonstrates how specialized dark and light versions of the main
+background fill and nav colors could be defined as varying lightness percentages
+of a green hue.
+
+```json
+{
+  "theme": {
+    "color": {
+      "hue-green": "120",
+      "fill": {
+        "dark": "hsl(var(--color-hue-green), 100%, 10%)",
+        "light": "hsl(var(--color-hue-green), 100%, 90%)"
+      },
+      "fill-secondary": {
+        "dark": "hsl(var(--color-hue-green), 100%, 20%)",
+        "light": "hsl(var(--color-hue-green), 100%, 80%)"
+      },
+      "nav-solid-background": "var(--color-fill)"
+    }
+  }
+}
+```
+
+![A screenshot of the green page background in dark mode](theme-screenshot-02.png)
+
+![A screenshot of the green page background in light mode](theme-screenshot-03.png)
+
+As a general rule, the default color properties provided by DocC-Render assumes
+a naming convention where "fill" colors are used for backgrounds and "figure"
+colors are used for foreground colors, like text.
+
+> Tip:
+> For a more complete example of a fully customized documentation website, you
+> can check out [this fork][3] of the DocC documentation that is using this
+> [theme-settings.json file][4].
+
+### Fonts
+
+By default, there are two main fonts used by DocC-Render: one for monospaced
+text in code voice and another for all other text content. With
+`theme-settings.json`, you can customize the CSS `font-family` lists for these
+however you choose.
+
+**Example**
+
+```json
+{
+  "theme": {
+    "typography": {
+      "html-font": "\"Comic Sans MS\", \"Comic Sans\", cursive",
+      "html-font-mono": "\"Courier New\", Courier, monospace"
+    }
+  }
+}
+```
+
+![A screenshot of a documentation page with custom fonts](theme-screenshot-04)
```

Comment:
The Comic Sans, grr… 😭😂 Also, it seems like the light/dark asset files are the wrong way around, the light asset as the `~dark` suffix.

Reply:
Lol, looks amazing, right? 🤭

I actually purposefully inverted the light/dark screenshots, because otherwise they kind of blend in to the background of the page, since they're screenshots of the page itself.